src/main/java/com/github/seratch/jslack/api/model/block/ActionsBlock.java

```diff
@@ -0,0 +1,23 @@
+package com.github.seratch.jslack.api.model.block;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.github.seratch.jslack.api.model.block.element.BlockElement;
+
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+/**
+ * https://api.slack.com/reference/messaging/blocks#actions
+ */
+@Data
+@EqualsAndHashCode(callSuper = true)
+@Builder
+public class ActionsBlock extends LayoutBlock {
+	private final String type = "actions";
```

Comment:
nit: in this project, we use soft tabs with 4 spaces.

Reply:
Do you mean indentation with spaces at 4? (I love it, but need confirmation to make Eclipse conform to your coding style)

Reply:
Yes, I do. It's also okay for me to apply my code formatter (I use IntelliJ IDEA's default one) when merging your changes.

Reply:
Looking at the classes my guess is typically 4 actual spaces. 👍

Reply:
I've updated the modified classes to use spaces instead of tabs.